src/main/java/Todo.java

```diff
@@ -0,0 +1,13 @@
+public class Todo extends Task {
+    protected String todo;
+    protected String category;
+
+    public Todo(String description) {
+        super(description);
+        this.category = "[T]";
+    }
+
```

Comment:
Can add "@Override" here to point out the method override.

Reply:
I will read up more regarding @override as I was unsure how it works! thanks